<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>630</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>630</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>630</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame7{background-color: rgb(243, 243, 243);}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frame4">
       <property name="minimumSize">
        <size>
         <width>770</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>770</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame4{border:5px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QChartView" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>530</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>760</width>
            <height>530</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>772</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#frame3{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
#frame3{background-color: rgb(243, 243, 243);}
</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(32, 74, 135);</string>
             </property>
             <property name="text">
              <string>freq：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="styleSheet">
              <string notr="true">background-color: rgba(114, 159, 207,80);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">color: rgb(32, 74, 135);</string>
             </property>
             <property name="text">
              <string>Vpp：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="styleSheet">
              <string notr="true">background-color: rgba(114, 159, 207,80);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">color: rgb(32, 74, 135);</string>
             </property>
             <property name="text">
              <string>Vmax：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgba(114, 159, 207,80);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">color: rgb(32, 74, 135);</string>
             </property>
             <property name="text">
              <string>Vmin：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_5">
             <property name="styleSheet">
              <string notr="true">background-color: rgba(114, 159, 207,80);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame8">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="maximumSize">
           <size>
            <width>115</width>
            <height>525</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#frame{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);
}
#frame{background-color: rgb(243, 243, 243);}
#pushButton
{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>2</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>15</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>111111</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(239, 41, 41);</string>
             </property>
             <property name="text">
              <string>RUN/STOP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>16</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(211, 215, 207);</string>
             </property>
             <property name="text">
              <string>触发电平</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDial" name="dial">
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>130</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QFrame" name="frame2_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgba(51, 161, 201,80);</string>
             </property>
             <layout class="QVBoxLayout" name="frame2">
              <item>
               <widget class="QRadioButton" name="radioButton_9">
                <property name="text">
                 <string>1V/div</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_10">
                <property name="text">
                 <string>0.1V/div</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_11">
                <property name="text">
                 <string>2mV/div</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_12">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(233, 185, 110);</string>
                </property>
                <property name="text">
                 <string>0.5V/div</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame1">
             <property name="styleSheet">
              <string notr="true">background-color: rgba(51, 161, 201,80);</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QRadioButton" name="radioButton">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>20ms/div</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_4">
                <property name="text">
                 <string>2us/div</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_2">
                <property name="text">
                 <string>100ns/div</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame5">
          <property name="minimumSize">
           <size>
            <width>115</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>115</width>
            <height>525</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#frame5{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);
}
#frame5{background-color: rgb(243, 243, 243);}
#pushButton_7
{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
#pushButton_2
{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(114, 159, 207);</string>
             </property>
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>109</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>109</width>
               <height>16</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(211, 215, 207);</string>
             </property>
             <property name="text">
              <string>Zoom</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDial" name="dial_3"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>111</width>
               <height>16</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(211, 215, 207);</string>
             </property>
             <property name="text">
              <string>stretch</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDial" name="dial_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(211, 215, 207);</string>
             </property>
             <property name="text">
              <string>POSITION_Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDial" name="dial_4"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(211, 215, 207);</string>
             </property>
             <property name="text">
              <string>POSITION_X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDial" name="dial_5"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(252, 233, 79);</string>
             </property>
             <property name="text">
              <string>CURSOR</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QFrame" name="frame6">
          <property name="styleSheet">
           <string notr="true">#frame6{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);
}
#frame6{background-color: rgb(243, 243, 243);}
#pushButton_3
{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
#pushButton_4
{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
#pushButton_5
{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
#pushButton_6
{border:1px solid;
border-radius:10px;
border-color: rgb(46, 52, 54);}
</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>109</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>109</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(173, 127, 168);</string>
             </property>
             <property name="text">
              <string>FFT</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>109</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>109</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(234, 22, 159);</string>
             </property>
             <property name="text">
              <string>SINGLE</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>109</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(115, 210, 22);</string>
             </property>
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>109</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>109</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(233, 185, 110);</string>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
